sprout/views/projects_id_metadata_id_data_update.py

```diff
@@ -51,11 +51,15 @@ def projects_id_metadata_id_data_update(
         # )
         # TODO: verify that database has been written to.
 
+        # update tables model with new data rows
+        table.data_rows = table.data_rows + count_rows(new_server_file)
+        table.save()
+
         context = {
-            "table_name": tables.name,
+            "table_name": table.name,
             "upload_success": True,
             "file_metadata": files,
-            "number_rows": count_rows(new_server_file),
+            "number_rows": table.data_rows,
```

Comment:
```suggestion
            "number_rows": new_rows_added,
```

this should be kept. It informs the user how many new rows have been added, not how many are currently there after the addition.

Reply:
Ah, you're right! My bad